File: in_Java/0171 Excel Sheet Column Number.java
class Solution {
    public int titleToNumber(String s) {
        int res = 0;
        int p = 0;
        for(int i = s.length()-1; i >= 0; i--){
            int cur = s.charAt(i) - 'A' + 1;
            res += cur * Math.pow(26,p++);
        }
        return res;
    }
}

File: in_Python/0171 Excel Sheet Column Number.py
class Solution(object):
    def titleToNumber(self, s):
        """
        :type s: str
        :rtype: int
        """
        res = 0
        power = 0
        for i in range(len(s)-1, -1, -1):
            c = s[i]
            n = ord(c) - ord('A') + 1
            res += n * pow(26, power)
            power += 1
        return res